app/main.py

```diff
@@ -16,4 +16,6 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [my_favourite_films, marks, collection_of_coins],
+    "immutable": [lucky_number, pi, one_is_a_prime_number, name, profile_info]}
```

Comment:
The `collection_of_coins` set is incorrectly categorized as 'immutable'. Sets in Python are mutable, so it should be placed under the 'mutable' category.

Reply:
set is mutable type.